sopel/config/core_section.py

```diff
@@ -1181,12 +1181,15 @@ def homedir(self):
 
     :default: ``PLAIN``
 
-    ``EXTERNAL`` is also supported, e.g. for using :attr:`client_cert_file` to
-    authenticate via CertFP.
+    Supported mechanisms are:
+        * ``PLAIN``, to authenticate by sending a plaintext password
+        * ``EXTERNAL``, to authenticate using a TLS client certificate
+          (see :attr:`client_cert_file`)
+        * ``SCRAM-SHA-256``, for password-based challenge-response authentication
```

Comment:
Indenting so much makes Sphinx generate a `<dl>`, with the `<ul>` and children nested inside a `<dd>`:

<details>
<summary>Dev Tools HTML view</summary>

![image](https://user-images.githubusercontent.com/164140/197937399-4134086a-3126-4734-ab8d-fd0905414e7a.png)

</details>

That's probably not what we want.

```suggestion
  * ``PLAIN``, to authenticate by sending a plaintext password
  * ``EXTERNAL``, to authenticate using a TLS client certificate
    (see :attr:`client_cert_file`)
  * ``SCRAM-SHA-256``, for password-based challenge-response authentication
```

Reply:
With an added bonus of that 81-char line now being 77